### DISCUSSION TITLE
Anura3D Calculation speed

### DISCUSSION THREAD

**Discussion by XJ-liu-9527** [Anura3D/Anura3D_OpenSource#258] [Q&A]
I built a model of 7000 units, as shown in Figure 1, including soil mass and pile. After the initial balance, the next load step calculation began, but the next step calculation was still not carried out several hours later. At the beginning, the task was calling CPU, but after a while the CPU usage was 0, as shown in Figure 2 and Figure 3. May I ask what the reason is?
Thank you all for your help！
![1-1](https://github.com/user-attachments/assets/ffce03cc-85ec-4b7c-adf6-13385098ccd2)
![1-2](https://github.com/user-attachments/assets/e153dcef-b6ba-4bd6-a137-babffd3dd2c9)                                              
                                         图1
![2](https://github.com/user-attachments/assets/8f23c12d-a2b2-4a68-83fe-16ec1080c320)
                                         图2
![3](https://github.com/user-attachments/assets/188a1f73-e18b-40ee-9954-36e0c84063c1)
                                         图3

**Comment by XJ-liu-9527** [comment]
**A few hours later, the calculation began; It's also a reminder that sometimes you have to wait.**
![image](https://github.com/user-attachments/assets/9e71ed5b-7af6-436c-bdf2-79b6df42c2a5)

**Comment by hjk1108** [comment]
I'm having the same issue. Has it been resolved?  How can I use CPU acceleration?

**Comment by panf7422-art** [comment]
May I ask how I can speed up the calculation process? Are there any friends who have solved this?
